### PR TITLE
Crosshairs

### LINKED ISSUE #170: Crosshair to show the other slice locations
This is something came from Jim and he mentioned that all other viewers have it. 

@annehaley fyi

### PR DESCRIPTION
**Pull request by annehaley**
This PR adds colored crosshairs to the VtkViewer components.

I worked with Forrest Li to use VTK internals for proper visualization of the crosshairs. The last two commits on this PR are the results of live pair programming sessions with Forrest to ensure that rendering is done properly.

Resolves #170

### CODE REVIEW

**Review by dchiquito — APPROVED** [review]
I am not well versed in the vtkjs stuff, but the vue stuff looks good to me.

File: client/src/components/VtkViewer.vue [hunk 1]
```diff
@@ -85,12 +104,31 @@ export default {
   mounted() {
     this.initializeView();
     this.initializeSlice();
+    this.updateCrosshairs();
+    this.renderSubscription = this.view.getInteractor().onRenderEvent(() => {
+      this.updateCrosshairs();
+    });
+    this.resizeObserver = new window.ResizeObserver((entries) => {
+      if (entries.length === 1) {
```
Comment (zachmullen) [inline client/src/components/VtkViewer.vue:112]: What are the `entries` here?

File: client/src/components/VtkViewer.vue [hunk 2]
```diff
@@ -241,7 +391,7 @@ export default {
   bottom: 0;
   left: 0;
   right: 0;
-  background: linear-gradient(#3a3a3a, #1d1d1d);
+  background: gray;
```
Comment (zachmullen) [inline client/src/components/VtkViewer.vue:394]: Is this change intentional or leftover from debugging?

**Review by zachmullen — APPROVED** [review]
LGTM, just a couple questions

File: client/src/components/VtkViewer.vue (hunk 1 above)
Comment (annehaley) [reply client/src/components/VtkViewer.vue:112]: Haha I had to ask Forrest the same thing. Part of the `ResizeObserver` [API](https://developer.mozilla.org/en-US/docs/Web/API/Resize_Observer_API) explains that the observer can observe multiple elements that got resized with the same action. This line was his recommendation.

File: client/src/components/VtkViewer.vue (hunk 2 above)
Comment (annehaley) [reply client/src/components/VtkViewer.vue:394]: good catch, I meant to change that to solid black so the image looks seamless against the backdrop. I think it looks cleaner than the linear gradient. What do you think?

File: client/src/components/VtkViewer.vue (hunk 1 above)
Comment (zachmullen) [reply client/src/components/VtkViewer.vue:112]: To make sure the logic matches the intent... if we receive one of these events that has multiple entries, we intentionally want to ignore it?

Comment (annehaley) [reply client/src/components/VtkViewer.vue:112]: Yes, we tell the observer to only observe one element per instance of the `VtikViewer` component. We don't want to execute this block for any entries other than the specified target `this.$refs.viewer`.
We have three instances of the `VtkViewer` component; without this `if`, this block would execute 3^2 times instead of 3 times.
